crates/corro-types/src/agent.rs

```diff
@@ -350,6 +351,23 @@ fn refactor_corro_members(tx: &Transaction) -> rusqlite::Result<()> {
     "#,
     )
 }
+fn create_sync_state(tx: &Transaction) -> rusqlite::Result<()> {
+    tx.execute_batch(
+        r#"
+        CREATE TEMP TABLE _variables (name TEXT PRIMARY KEY, var TEXT);
+        INSERT INTO _variables VALUES ('current_ts', strftime('%Y-%m-%dT%H:%M:%S.000000000Z', CURRENT_TIMESTAMP));
+
+        UPDATE __corro_bookkeeping SET ts = (SELECT var FROM _variables WHERE name = 'current_ts') WHERE ts IS NULL;
```

Comment:
also: should this only update the current node? the current node is **the source of truth for its own clears**.

Reply:
> Interesting! Is that faster than just using strftime directly instead of a temporary table?

I don't think so. I am using a temp table because I want to put the same value in `__corro_sync_state` for this note. I guess its a big deal if they are slightly diffferent